gocourse05/camera/day_test.go

```diff
@@ -0,0 +1,58 @@
+package camera
+
+import (
+	"testing"
+)
+
+func TestDayCameraRetrieveData(t *testing.T) {
+	t.Run("Found", func(t *testing.T) {
+		c := NewDayCamera(1, "Canon", []Data{NewDayCameraData(1, "Коза", "Побігла вліво")})
+
+		data, err := c.RetrieveData()
+		if err != nil {
+			t.Errorf("RetrieveData() should not have error: %s", err)
```

Comment:
Краще робити [`t.Fatalf`](https://pkg.go.dev/testing#T.Fatalf) замість `t.Errorf`, оскільки немає сенсу перевіряти далі:
```suggestion
			t.Fatalf("RetrieveData() should not have error: %s", err)
```